util/include/openssl_utils.hpp

```diff
@@ -0,0 +1,157 @@
+// Concord
+//
+// Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+#pragma once
+
+#include <string>
+#include <memory>
+#include <fstream>
+
+#include <openssl/bio.h>
+#include <openssl/ec.h>
+#include <openssl/pem.h>
+#include <openssl/x509.h>
+#include <openssl/evp.h>
+
+#include "crypto_interface.hpp"
+#include "crypto_utils.hpp"
+
+using std::pair;
+using std::make_pair;
+using std::string;
+using std::unique_ptr;
+using std::ofstream;
+using concord::util::crypto::KeyFormat;
+using concord::util::cryptointerface::ISigner;
+using concord::util::cryptointerface::IVerifier;
+
+namespace concord::util::openssl_utils {
+
+constexpr static size_t SIG_LENGTH{64};
```

Comment:
Please rename to indicate what signature this is.
Assuming that this is EdDSA, I added a header with all of the relevant constant in my branch.

Reply:
Done.